validator/client/validator_aggregate_test.go

```diff
@@ -78,3 +79,26 @@ func TestWaitForSlotTwoThird_WaitCorrectly(t *testing.T) {
 		t.Errorf("Wanted %d time for slot two third but got %d", twoThirdTime, currentTime)
 	}
 }
+
+func TestAggregateAndProofSignature_CanSignValidSignature(t *testing.T) {
+	validator, m, finish := setup(t)
+	defer finish()
+
+	m.validatorClient.EXPECT().DomainData(
+		gomock.Any(), // ctx
+		gomock.Any(), // epoch
```

Comment:
Do you know the epoch value? That would be preferred over gomock.Any()

Reply:
Good point. Using:
```
		&ethpb.DomainRequest{Epoch: 0, Domain: params.BeaconConfig().DomainAggregateAndProof[:]},
```